Directory.Build.props

```diff
@@ -12,8 +12,13 @@
     <DebugType>Portable</DebugType>
     <LangVersion>9.0</LangVersion>
     <HighEntropyVA>true</HighEntropyVA>
-    <EnableSourceLink Condition="$([MSBuild]::IsOSPlatform('osx'))">false</EnableSourceLink>
-    <EnableSourceControlManagerQueries>$(EnableSourceLink)</EnableSourceControlManagerQueries>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
This is also above

Reply:
indeed...

Reply:
One part of me want's to sort them, another part want's to break them into different property groups (not sure I can do clear cut by nuget related/ vs everything else), but there also third part which is just lazy, and whispering into my ear: "No one cares". If you think any of approaches worth it, I'll do it, I just not sure is it really worth it or not.